docs/docs/tutorial/part-7/index.mdx

```diff
@@ -0,0 +1,492 @@
+    ![A screenshot of the Unsplash home page](./unsplash.png)
+
+2. When you've found a photo that you like, download it and add it to subdirectory for one of your blog posts. Continue downloading photos until you have a different hero image for each post.
+
+3. Next, add some additional frontmatter fields to each of your blog posts:
+    * `hero_image`: the relative path to the hero image file for that post
+    * `hero_image_alt`: a short description of the image, to be used as alternative text for screen readers or in case the image doesn't load correctly
+    * `hero_image_credit_text`: the text to display to give the photographer credit for the hero image
+    * `hero_image_credit_link`: a link to the page where your hero image was downloaded from
+
+```mdx:title=blog/my-first-post/index.js
+---
+title: "My First Post"
+date: "2021-07-23"
+// highlight-start
+hero_image: "./christopher-ayme-ocZ-_Y7-Ptg-unsplash.jpg"
+hero_image_alt: "A gray pitbull relaxing on the sidewalk with its tongue hanging out"
+hero_image_credit_text: "Christopher Ayme"
+hero_image_credit_link: "https://unsplash.com/photos/ocZ-_Y7-Ptg"
+// highlight-end
+---
+
+...
+```
+
+```mdx:title=blog/another-post/index.js
+---
+title: "Another Post"
+date: "2021-07-24"
+// highlight-start
+hero_image: "./anthony-duran-eLUBGqKGdE4-unsplash.jpg"
+hero_image_alt: "A grey and white pitbull wading happily in a pool"
+hero_image_credit_text: "Anthony Duran"
+hero_image_credit_link: "https://unsplash.com/photos/eLUBGqKGdE4"
+// highlight-end
+---
+
+...
+```
+
+```mdx:title=blog/yet-another-post/index.js
+---
+title: "Yet Another Post"
+date: "2021-07-25"
+// highlight-start
+hero_image: "./jane-almon-7rriIaBH6JY-unsplash.jpg"
+hero_image_alt: "A white pitbull wearing big googly-eye glasses"
+hero_image_credit_text: "Jane Almon"
+hero_image_credit_link: "https://unsplash.com/photos/7rriIaBH6JY"
+// highlight-end
+---
+
+...
+```
+
+
+Now that your hero images are set up, it's time to connect them to the data layer so you can pull them into your blog post page template.
+
+## Install and configure `gatsby-transformer-sharp`
+
+In order to use the `GatsbyImage` component, you'll need to add the `gatsby-transformer-sharp` transformer plugin to your site.
+
+When Gatsby adds nodes to the data layer at build time, the `gatsby-transformer-sharp` plugin looks for any File nodes that end with an image extension (like `.png` or `.jpg`) and creates an ImageSharp node for that file.
+
+![A diagram showing how ImageSharp nodes get created from File nodes in the data layer](./data-layer-with-imagesharp-nodes.png)
+
+1. In the terminal, run the following command to install `gatsby-transformer-sharp`:
+    ```shell
+    npm install gatsby-transformer-sharp
+    ```
+
+2. Add `gatsby-transformer-sharp` to the `plugins` array in your `gatsby-config.js` file.
+
+```js:title=gatsby-config.js
```

Comment:
Should probably be intended like code block for 1.?

Reply:
Good catch. Fixed!